src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -101,4 +101,51 @@ private CallbackValidations() {
     static Validation<String, List<Map<String, Object>>> hasAScannedRecord(CaseDetails theCase) {
         return scannedDocumentValidator.validate(theCase);
     }
+
+    @Nonnull
+    static Validation<String, String> hasValidCombinationOfEventIdAndClassification(
```

Comment:
Please suggest if you think this validation could be done in a better way

Reply:
Maybe it should be `Validation<String, Void>`, as we don't want to return any specific value? We'll probably have to validate these two before the existing validation, so that if this fails, we don't even validate the rest.